lib/metasploit/framework/mssql/client.rb

```diff
@@ -282,7 +291,15 @@ def mssql_login(user='sa', pass='', db='', domain_name='')
             # Packet header and total length including header
             pkt = "\x10\x01" + [pkt.length + 8].pack('n') + [0].pack('n') + [1].pack('C') + "\x00" + pkt
 
-            resp = mssql_send_recv(pkt)
```

Comment:
It might be nicer if ``mssql_send_recv`` was modified to handle SSL directly within the mixin. This would cut back on  the number of ``if`` blocks here.